renormalizer/model/mlist.py

```diff
@@ -351,10 +337,10 @@ def e_idx(idx):
         dipole = {}
         for imol, mol in enumerate(mol_list):
             if mol_list.scheme < 4:
-                dipole[(f"e_{e_idx(imol)}", )] = mol.dipole
+                dipole[(f"e_{imol}", )] = mol.dipole
             elif mol_list.scheme == 4:
-                dipole[(f"e_{e_idx(imol)}", "e_0")] = mol.dipole
-                dipole[("e_0", f"e_{e_idx(imol)}")] = mol.dipole
+                dipole[(f"e_{imol}", "e_0")] = mol.dipole
```

Comment:
It is not correct here. The dipole matrix is between vacuum and e_{imol}. 
I think we can only keep `dipole[(f"e_{imol}", )] = mol.dipole` now and come back again when we deal with this kind of "transition" problem someday.